src/dev/flang/fe/SourceModule.java

```diff
@@ -405,7 +405,7 @@ part of the (((inner features))) declarations of the corresponding
 
                     Files.list(d._dir)
                       .filter(p -> isValidSourceFile(p))
-                      .sorted()
+                      .sorted(Comparator.comparing(p -> p.toString(), String.CASE_INSENSITIVE_ORDER))
```

Comment:
Sorry, I am getting a bit picky: This will produce an ambiguous order on systems that are case-sensitive in case there a file names that differ only in case. So maybe better just do
```suggestion
                      .sorted(Comparator.comparing(p -> p.toString()))
```
which I think should work for Windows as well since there will be no file names that differ by case.

Reply:
Yes, that's better..